terms.sample.yaml

```diff
@@ -0,0 +1,20 @@
+master_version: "master_1_1"
+docs:
+  terms_of_service:
+    version: "2.0"
+    langs:
+      en:
```

Comment:
Any reason that this doesn't follow the same structure as the response to the terms API?

Reply:
It makes it easier to parse into objects this way (the version isn't mixed in with all the langs)